packages/@aws-cdk/aws-codecommit/test/integ.codecommit-code-asset.ts

```diff
@@ -0,0 +1,13 @@
+import * as cdk from '@aws-cdk/core';
+import * as codecommit from '../lib';
+import { Code } from '../lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-codecommit-assets');
```

Comment:
These should have different names from each other:

```suggestion
const stack = new cdk.Stack(app, 'aws-cdk-codecommit-repo-contents-assets');
```